shared/js/dispatcher.js

```diff
@@ -92,7 +92,7 @@ loop.Dispatcher = (function() {
       this._active = true;
 
       if (this._debug) {
-        console.log("[Dispatcher] Dispatching action", action);
+        console.log("[Dispatcher] Dispatching action", action, action.name);
```

Comment:
The name is already in the action object, so I don't see the need for this? Or are you trying to compensate for the browser console in e10s mode?

If you're compensating, then I'd suggest putting the action.name first before the action object.